### PR TITLE
chore: implement arbitrary comparison for BLS keys

### PR DESCRIPTION
**Pull request by mrain**
## Description

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (main)
- [ ] Linked to GitHub issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [ ] Wrote unit tests
- [ ] Updated relevant documentation in the code
- [ ] Added a relevant changelog entry to the `Pending` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the GitHub PR explorer

### CODE REVIEW

File: primitives/src/signatures/bls_over_bn254.rs [hunk 1]
```diff
@@ -250,6 +260,16 @@ impl PartialEq for VerKey {
     }
 }
 
+// An arbitrary comparison for VerKey. Doesn't mean anything.
+impl PartialOrd for VerKey {
+    fn partial_cmp(&self, other: &Self) -> Option<core::cmp::Ordering> {
+        self.0
+            .into_affine()
```
Comment (ggutoski) [inline primitives/src/signatures/bls_over_bn254.rs:267]: This is a nontrivial computation, no? Is there no way to avoid it?

**Review by ggutoski — DISMISSED** [review]
Approved so as not to slow progress but please see my comment

**Comment by philippecamacho** [comment]
Should we add a test?

**Comment by mrain** [comment]
> Should we add a test?

test added in a45fcb5

File: primitives/src/signatures/bls_over_bn254.rs (hunk 1 above)
Comment (mrain) [reply primitives/src/signatures/bls_over_bn254.rs]: Issue created here: https://github.com/EspressoSystems/jellyfish/issues/457

**Review by philippecamacho — DISMISSED** [review]
A minor comment. LGTM

**Review by ggutoski — APPROVED** [review]

**Pull request merged**